README.md

```diff
@@ -4,7 +4,49 @@ Copyright Debezium Authors. Licensed under the [Apache License, Version 2.0](htt
 
 This repository contains multiple examples for using Debezium, e.g. configuration files, Docker Compose files, OpenShift templates.
 
-
 ## Getting Started
 
 For getting started please check the [tutorial example](./tutorial).
+
+## Examples
+
+* [Apache Pulsar](./apache-pulsar): Streaming **Postgres** database changes to **Apache Pulsar**
+* [Audit Logs](./auditlog): Building Audit Logs with Change Data Capture
+* [Cache Invalidation](./cache-invalidation): How Debezium can be used to invalidate items in the JPA 2nd level cache after external data changes
+* [Camel - pipelines](./camel-component): Building an Apache Camel pipeline that captures **Postgres** database changes
+* [Camel - Kafka Connect](./camel-kafka-connect): How to use the Camel Kafka Connect component with Debezium
+* [Cloud Events](./cloudevents): How to use cloud events defined in Json with Debezium
+* [Database Activity Monitoring](./db-activity-monitoring): How to use Debezium for comprehensive database activity logging and analysis
+* [Debezium - End-to-end demo](./end-to-end-demo): End-to-end demo using MySQL as database and Kafka Connect
+* [Debezium - Failover](./failover): How Debezium recovers after a database failure
+* [Debezium - Monitoring](./monitoring): Monitoring a Debezium instance
+* [Debezium - Auto-Creation of Topics](./topic-auto-create): Auto-creating Debezium change data topics
+* [Debezium - Outbox Pattern](./outbox): Implement the "outbox pattern", an approach for letting services communicate in an asynchronous and reliable fashion
+* [Debezium - Saga Pattern](./saga): How to implement the [Saga pattern](https://microservices.io/patterns/data/saga.html) for realizing distributed transactions across multiple microservices
+* [Debezium - Testing](./testcontainers): How to implement an integration test for your CDC set-up using [Testcontainers](https://www.testcontainers.org/)
+* [Debezium - User Interface](./ui-demo): How to deploy Debezium UI
```

Comment:
@jpechane  There was another PR where I suggested to not list the Debezium UI here because of it current outdated state. Should we drop this line in another PR?

Reply:
@rk3rn3r Makes sense. Should we remove it from the list or should we remove the example completely?